src/nvim/ex_cmds.c

```diff
@@ -5535,6 +5535,9 @@ static buf_T *show_sub(exarg_T *eap, pos_T old_cusr,
 
   // We keep a special-purpose buffer around, but don't assume it exists.
   buf_T *preview_buf = bufnr ? buflist_findnr(bufnr) : 0;
+  // disable :leftabove/botright modifiers
+  // (especially ones that conflict with our win_split() call below)
```

Comment:
Is that the right thing to do though? Can't we somehow pass the modifier to our split call, so it's not ignored, but used properly?

Reply:
Yeah - out of the possible values that a user could give though, I don't know how popular/used they would be, especially as dropping the modifier is the existing behaviour with `:tab`, which leads me to think that not dropping the window position modifiers was just an oversight in the original change. What do you think?

Reply:
Makes sense, let's try this way for now. Can revisit later if needed.